python/ray/tests/BUILD

```diff
@@ -131,6 +131,7 @@ py_test_module_list(
     "test_args.py",
     "test_asyncio_cluster.py",
     "test_asyncio.py",
+    "test_concurrency_group.py",
```

Comment:
So before this PR, the test was always disabled?

Reply:
Yes. The test needs to be added to bazel so it can run.